app/screens/gallery/image_renderer/transformer.tsx

```diff
@@ -524,6 +530,27 @@ const ImageTransformer = (
         };
     }, []);
 
+    let element;
+    if (isSvg) {
+        element = (
+            <SvgUri
+                uri={imageSource.uri!}
+                style={styles.svg}
+                width={Math.min(targetDimensions.width, targetDimensions.height)}
+                height={Math.min(targetDimensions.width, targetDimensions.height)}
```

Comment:
Should we check for a maximum size? (i.e. imagine a svg of 10.000x10.000)

Reply:
we can check for it but not really here, it should be in the function that calculates the dimension based on the source